README.md

```diff
@@ -4,7 +4,11 @@
 
 SignalR plugin for Vue 3.
 
-## Installation
+## Quick Start
+
+For more detailed instructions, check out the wiki at <https://github.com/quangdaon/vue-signalr/wiki>.
```

Comment:
In this context, I might put this in a blockquote or italics since it's kind of a "callout"